index.js

```diff
@@ -152,6 +153,22 @@ Parser.prototype = {
     return this.initBlock(line, []);
   },
 
+  runPlugin: function(context, tok) {
+    var rest = [this];
+    for (var i = 2; i < arguments.length; i++) {
+      rest.push(arguments[i]);
+    }
+    var pluginContext;
+    for (var i = 0; i < this.plugins.length; i++) {
+      var plugin = this.plugins[i];
+      if (plugin[context] && plugin[context][tok.type]) {
+        if (pluginContext) throw new Error('Multiple plugin handlers found for context ' + JSON.stringify(context) + ', token type ' + JSON.stringify(tok.type));
```

Comment:
I could move this check to the constructor instead of having to do it every run.